### PR TITLE
remove lab identifiers from plugin identifiers

### PR DESCRIPTION
**Pull request by mschrimpf**
Change identifiers from `<lab-pi-lastname>.<firstauthor-lastname><year>` to `<firstauthor-lastname><year>`. This affects data and benchmark plugins; metrics and models use a different naming scheme.

Motivation: in all the years of brain-score, I have never heard of anyone refer to data or benchmark with the lab prefix. On the website, we even strip the prefix to save space. This PR names the plugins in more colloquial terms.

Note that this leads to some discrepancy:
* the S3 files still have the lab identifier prefix
* this also means that while e.g. stimulus sets will be loaded with just the `<firstauthor-lastname><year>` identifier, the `stimulus_set.identifier` will still include the lab prefix

### CODE REVIEW

**Comment by mschrimpf** [comment]
verified that no more labs show up in library plugin identifiers with the following script (modified from [`parse_plugin_changes.py`](https://github.com/brain-score/core/blob/main/brainscore_core/plugin_management/parse_plugin_changes.py)):
```python
def get_plugin_ids(plugin_type: str) -> List[str]:
    plugin_type_root = Path(__file__).parent.parent / 'brainscore_vision' / plugin_type
    plugin_ids = []

    for plugin_dirname in plugin_type_root.iterdir():
        if not plugin_dirname.is_dir() or str(plugin_dirname).endswith('__pycache__'): continue

        init_file = plugin_dirname / '__init__.py'
        with open(init_file) as f:
            registry_name = plugin_type.strip(
                's') + '_registry'  # remove plural and determine variable name, e.g. "models" -> "model_registry"
            plugin_registrations = [line for line in f if f"{registry_name}["
                                    in line.replace('\"', '\'')]
            for line in plugin_registrations:
                result = re.search(f'{registry_name}\[.*\]', line)
                identifier = result.group(0)[len(registry_name) + 2:-2]  # remove brackets and quotes
                plugin_ids.append(identifier)

    return plugin_ids


def __main__():
    for plugin_type in ['data', 'metrics', 'benchmarks', 'models']:
        plugin_ids = get_plugin_ids(plugin_type)
        ids_with_dot = [plugin_id for plugin_id in plugin_ids if '.' in plugin_id]
        print(ids_with_dot)

```

**Review by kvfairchild — CHANGES_REQUESTED** [review]
[Travis tests](https://app.travis-ci.com/github/brain-score/brain-score/builds/267918408) need resolving; they are (correctly) marked as failing on the PR public build but also need review in the (incorrectly marked as passing) private build. I've opened a ticket with Travis support about the incorrect pass (see Slack for further details).

I don't think any of the [Jenkins failures](http://braintree.mit.edu:8080/job/unittest_brainscore/1226/) are actually stemming from  this PR but it would be nice to track those down so we can merge this with a green Jenkins check as well.

File: brainscore_vision/benchmarks/objectnet/__init__.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
 from brainscore_vision import benchmark_registry
 from .benchmark import Objectnet
 
-benchmark_registry['katz.BarbuMayo2019-top1'] = Objectnet
\ No newline at end of file
+benchmark_registry['BarbuMayo2019-top1'] = Objectnet
```
[the remark this answers is not in the document]
Comment (mschrimpf) [reply brainscore_vision/benchmarks/objectnet/__init__.py:4]: you're right, will update

File: brainscore_vision/data/imagenetslim15000/test.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import pytest
+
+from brainscore_vision import load_dataset
+
+
+@pytest.mark.private_access
+def test_existence():
+    assert load_dataset('ImageNetSlim15000') is not None
```
[the remark this answers is not in the document]
Comment (mschrimpf) [reply brainscore_vision/data/imagenetslim15000/test.py:8]: [right here](https://github.com/brain-score/brain-score/blob/43a51acc41093c5a790fdbc1b75c6694afeeecaa/brainscore_vision/data/imagenetslim15000/__init__.py#L5)

[the remark this answers is not in the document]
Comment (mschrimpf) [reply brainscore_vision/data/imagenetslim15000/test.py:8]: but it should be `load_stimulus_set`

**Pull request merged**